docs/SIMPLEX.md

```diff
@@ -50,7 +50,7 @@ You can use SimpleX with your own servers and still communicate with people usin
 
 SimpleX platform uses an open protocol and provides SDK to create chat bots, allowing implementation of services that users can interact with via SimpleX Chat apps – we are really looking forward to see what SimpleX services can be built.
 
-If you are considering developing with SimpleX platform, whether to build chat bot service for SimpleX apps users or to integrate SimpleX Chat library in your mobile apps, please get in touch for any advice and support.
```

Comment:
We could link to our address here to reduce the friction to getting in touch. Any suggestions on what to link (if at all)?

Reply:
yes, thought about it for a second... I think it's ok as is, it's very easy to find out